mapiproxy/servers/default/nspi/dcesrv_exchange_nsp.h

```diff
@@ -41,6 +41,10 @@
 #endif
 #endif
 
+/* The IS_WINUNICODE_CODEPAGE  macro checks that the code page in the in.pStat value is CP_WINUNICODE (equivalent to the CP_UNICODE macro).*/
+#define IS_WINUNICODE_CODEPAGE(r) (r->in.pStat->CodePage == CP_UNICODE)
+#define CONTAINER_EXISTS(r) ((!r->in.pStat->ContainerID) || (emsabp_tdb_lookup_MId(emsabp_ctx->tdb_ctx, r->in.pStat->ContainerID)) == true)
```

Comment:
It would not be better to add emsabp_ctx to the macro?. This check is used 4 or 5 times accross various methods